modules/renter/renter.go

```diff
@@ -281,15 +284,17 @@ func (r *Renter) SetSettings(s modules.RenterSettings) error {
 	if err != nil {
 		return err
 	}
-	// Set bandwidth limits.
+	// Set ratelimit
 	if s.MaxDownloadSpeed < 0 || s.MaxUploadSpeed < 0 {
 		return errors.New("download/upload rate limit can't be below 0")
 	}
-	// Set ratelimit
 	if s.MaxDownloadSpeed == 0 && s.MaxUploadSpeed == 0 {
-		ratelimit.SetLimits(0, 0, 0)
+		r.hostContractor.SetRateLimits(0, 0, 0)
 	} else {
-		ratelimit.SetLimits(s.MaxDownloadSpeed, s.MaxUploadSpeed, 4*4096)
+		// TODO: In the future we might want the user to be able to configure the
+		// packetSize using the API. For now the sane default is modules.SectorSize
+		// if the user wants to limit the connection.
+		r.hostContractor.SetRateLimits(s.MaxDownloadSpeed, s.MaxUploadSpeed, 4*4096)
```

Comment:
sane default is not modules.SectorSize, it's just 16 KiB

Reply:
fixed